CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.2.0] - 2021-06-30
+### Added
+- Adding TokenListController to fetch the token list from token services API ([#478](git+https://github.com/MetaMask/controllers/pull/478))
+- Feature/use @ethereumjs/tx ([#489](git+https://github.com/MetaMask/controllers/pull/489))
+- Standardize repository ([#486](git+https://github.com/MetaMask/controllers/pull/486))
+- Bump @metamask/contract-metadata from 1.25.0 to 1.26.0 ([#492](git+https://github.com/MetaMask/controllers/pull/492))
```

Comment:
The two contract metadata updates can be consolidated into one entry. They should be in the same category as well.

Reply:
done!